content/machinelearning/naive_bayesian_classifier.Jmd

```diff
@@ -130,9 +130,9 @@ select!(leftovers, Not(:cultivar))
 first(leftovers, 3)
 ```
 
-Let's take the first row. We want to get the probability of every variable being
```

Comment:
I found this sentence a little bit confusing, so I suggested "observation" instead of "variable" because that made sense in my head, but if that's not the right info, then I take the suggestion back :laughing: